presets/mainnet/deneb.yaml

```diff
@@ -4,5 +4,5 @@
 # ---------------------------------------------------------------
 # `uint64(4096)`
 FIELD_ELEMENTS_PER_BLOB: 4096
-# `uint64(2**2)` (= 4)
-MAX_BLOBS_PER_BLOCK: 4
+# `uint64(2**4)` (= 16)
+MAX_BLOBS_PER_BLOCK: 16
```

Comment:
this should remain 4: that already represents a significant increase in traffic on the gossipsub network